opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -813,6 +839,12 @@ namespace Opm {
              const double dt)
     {
 
+        Opm::DeferredLogger local_deferredLogger;
+        if (this->glift_debug) {
+            std::ostringstream os;
+            os << "assemble() : iteration = " << iterationIdx;
+            gliftDebug(os, local_deferredLogger);
```

Comment:
With the `libfmt`library which I have shoehorned in you could do:
```
#include <fmt/format.h>
...
const std::string msg = fmt::format("assemble() : iteration {}" , iterationIdx);
```
Pick your poison

Reply:
Thanks this looks much better! I have updated the code to use `fmt::format()`